3Dmol/glviewer.js

```diff
@@ -2575,6 +2576,163 @@ $3Dmol.GLViewer = (function() {
             return s;
         };
         
+        /**
+         * Create volumetric renderer for volumetricData
+         * @function $3Dmol.GLViewer#addVolumetricRenderer
+         * @param {$3Dmol.VolumeData} data - volumetric data
+         * @param {IsoSurfaceSpec} spec - Shape style specification
```

Comment:
I would think this would have its own specification?  For example the transferfn.

Reply:
I don't fully understand what you mean by that, can you elaborate on that?